src/localized-text/LocalizedAppearanceEditor.component.js

```diff
@@ -260,6 +260,7 @@ class LocalizedTextEditor extends React.Component {
                         value={this.state.locale || ''}
                         floatingLabelText={i18n.t('Select language')}
                         onChange={this.handleChange}
+                        style={{ width: '100%' }}
```

Comment:
I've updated this field to be 100% width (rather than default width), because the new format `name - displayName` means that the selections were often wider than the input.